hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -221,7 +221,7 @@ hedera.prefetch.threadPoolSize=4
 hedera.prefetch.codeCacheTtlSecs=600
 utilPrng.isEnabled=true
 tokens.autoCreations.isEnabled=true
-virtualdatasource.jasperdbToMerkledb=false
+virtualdatasource.jasperdbToMerkledb=true
```

Comment:
If we enable the property in the default `bootstrap.properties` file, do we need to duplicate it in secondary files intended to overwrite it?

Reply:
I don't know the policy behind config files under hedera-node/configuration. Should they only contain non-default values? I find it convenient to list data on disk / MerkleDb related properties in all bootstrap.properties, even if some of them are default. Eventually, we may want to remove the defaults, e.g. when `jasperdbToMerkledb` is dropped altogether with JasperDB.